docs/operating-scylla/nodetool-commands/compact.rst

```diff
@@ -6,7 +6,7 @@ Compaction is an optimization that reduces the cost of IO and CPU over time by m
 
 By default, major compaction runs on all the ``keyspaces`` and tables.
 Major compactions will take all the SSTables for a column family and merge them into a **single SSTable per shard**.
-If a keyspace is provided, the compaction will run on all of the tables within that keyspace. If one or more tables are provided as command-line arguments, the compaction will run on all tables.
+If a keyspace is provided, the compaction will run on all of the tables within that keyspace. If one or more tables are provided as command-line arguments, the compaction will run on those tables.
```

Comment:
how about adding "only" like:

> If one or more tables are provided as command-line arguments, the compaction will only run on those tables.

Reply:
+1
Adding "only" would make it even clearer.